Update README.md for Demo (#577)
Adyen/adyen-ios: Demo/README.md

# Demo

Sample project to try out iOS Drop-in and Components integrations.

![DropIn preview](Screenshots/dropin-ios.jpg)

## How to start

> To start testing SDK, you need to open `Adyen.xcodeproj` in Xcode and provide the following information to `Demo/Configuration.swift`

Reviewer: This section sounds like required configuration for running the Demo. It doesn't say anything about how to run it. How about renaming to something like "Configuration"?

Author: Renamed to "Configuration"

### UIKit Demo

This is the basic UI for UIKit integration.

> You can run it by selecting `AdyenUIHost` target in xcode.

Reviewer: ```suggestion
Run the UIKit Demo by selecting `AdyenUIHost` as the target in Xcode.
```
Is it enough to select the target, or do they need to so other things?

### SwiftUI Demo

This is the basic UI for SwiftUI integration.

> You can run it by selecting `AdyenSwiftUIHost` target in xcode.

Reviewer: ```suggestion
Run the SwiftUI Demo it by selecting `AdyenSwiftUIHost` as the target in Xcode.
```
Same as above, do users need to do anything else after selecting the target?

Author: I guess it should be clear from this point how to run the project.
